GoogleForCreators/web-stories-wp
# Enable clicking through masked elements' empty frame area

## Summary
Moves the click-handler directly to the path inside the mask to enable clicking for direct clicks on the visible element only.


## Relevant Technical Choices



## To-do



## User-facing changes
The user can now click "through" the area inside the selection frame that is not covered by the element in case of masked elements.



## Testing Instructions
1. Add some elements (e.g. an image, a text) and place these to the upper left and upper right corner.
2. Add a triangle shape.
3. Extend it so that its blue selection box would cover the whole Page.
4. Verify that you can still select the image and the text by clicking on them. (Previously it always selected the triangle)


---



Fixes #2038
